twopaneview/examples/README.md

```diff
@@ -0,0 +1,46 @@
+# TwoPaneView
+
+## Status
+
+- Android:
+  - 10+
+- react-native:
+  - supported versions "<strong>&gt;= 0.60.5</strong>"
+
+## Installation
+
+### Please refer to the react-native-dualscreen page to install the latest version of DualScreenInfo first.
+
+- npm install react-native-twopaneview
+
+## API
+
+[Documentation](https://github.com/react-native-community/discussions-and-proposals/issues/197)
+- `Auto` 
+![](https://github.com/microsoft/react-native-dualscreen/raw/master/twopaneview/examples/paneModesDemo/auto.JPG)
+- `Single`
+![](https://github.com/microsoft/react-native-dualscreen/raw/master/twopaneview/examples/paneModesDemo/single.JPG)
+- `Dual`
+![](https://github.com/microsoft/react-native-dualscreen/raw/master/twopaneview/examples/paneModesDemo/dual.JPG)
+- `panePriority='pane2'`
+![](https://github.com/microsoft/react-native-dualscreen/raw/master/twopaneview/examples/paneModesDemo/priority2.JPG)
+- `onModeChanged={()=>{console.log('onModeChanged');}`
+
+## How to run sample app
+
+### 1. npm install
```

Comment:
People should clone the repo to run the examples

Reply:
Right. Step 0 is now to clone the repo